packages/sources/svelte/src/components/navigation/VtmnTabs/VtmnTabsItem.svelte

```diff
@@ -0,0 +1,29 @@
+<script>
+  import VtmnBadge from './../../indicators/VtmnBadge/VtmnBadge.svelte';
+  import VtmnIcon from './../../../guidelines/iconography/VtmnIcon/VtmnIcon.svelte';
+  export let href = '#';
+  export let badge = undefined;
```

Comment:
Please change it to `badgeValue` like on `VtmnChip`

Reply:
Done